plugins/parsers/prometheusremotewrite/parser.go

```diff
@@ -3,19 +3,16 @@ package prometheusremotewrite
 import (
 	"errors"
 	"fmt"
-	"math"
-	"time"
 
-	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/prompb"
 
 	"github.com/influxdata/telegraf"
-	"github.com/influxdata/telegraf/metric"
 	"github.com/influxdata/telegraf/plugins/parsers"
 )
 
 type Parser struct {
-	DefaultTags map[string]string
+	DefaultTags   map[string]string
+	MetricVersion int `toml:"prometheus_metric_version"`
```

Comment:
Please list the user-configurable things at the top
```suggestion
	MetricVersion int `toml:"prometheus_metric_version"`
	DefaultTags   map[string]string
```